bigquery_etl/shredder/config.py

```diff
@@ -255,6 +255,17 @@ def fields(self) -> tuple[str, ...]:
         DeleteSource(table="firefox_ios.deletion_request", field=GLEAN_CLIENT_ID),
         DeleteSource(table="fenix.deletion_request", field=GLEAN_CLIENT_ID),
     ),
+    DeleteTarget(
+        table="telemetry_derived.rolling_cohorts_v2",
+        field=(CLIENT_ID, CLIENT_ID, CLIENT_ID, CLIENT_ID, CLIENT_ID, CLIENT_ID),
+    ): (
+        DESKTOP_SRC,
+        DeleteSource(table="firefox_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="fenix.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="klar_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="focus_ios.deletion_request", field=GLEAN_CLIENT_ID),
+        DeleteSource(table="klar_android.deletion_request", field=GLEAN_CLIENT_ID),
```

Comment:
Should this also have `focus_android`?
```suggestion
        DeleteSource(table="focus_android.deletion_request", field=GLEAN_CLIENT_ID),
        DeleteSource(table="klar_android.deletion_request", field=GLEAN_CLIENT_ID),
```

Reply:
Thank you for catching that, pushed another commit